### PR TITLE
Check-Win Conditions

### PR DESCRIPTION
**Pull request by Jkhets**

### CODE REVIEW

File: api/Objects/player.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+export default class player{
+    constructor(id, username, score,){
+        this.id = id;
+        this.username = username;
+        this.score = score;
+        this.cards = [];
+    }
+    
+    get Cards(){
```
Comment (JoshAdamPowell) [inline api/Objects/player.js]: should change this method to lowercase `cards()` for consistency

File: api/Objects/player.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    get points(){
+        return this.score;
+    }
+
+    addCard(card){
+        this.cards.push(card);
+    }
+
+    playCard(pick, card){
```
Comment (JoshAdamPowell) [inline api/Objects/player.js]: Can you add a comment on what this code is doing? it looks a bit confusing to me!

File: api/tests/roundService.test.js [hunk 3]
```diff
@@ -72,4 +73,26 @@ test('if end of player list, reset to 0', () => {
             score: 8,
             ID: 1
         });
-})
\ No newline at end of file
+})
+
+test('check if the user has won the game', () => {
```
Comment (JoshAdamPowell) [inline api/tests/roundService.test.js]: test names should say what they're expecting so it's more obvious when it fails whats gone wrong, so something like "If user has 10 points expect checkwin to return true"

File: api/Objects/player.js (hunk 2 above)
Comment (Jkhets) [reply api/Objects/player.js]: Oops, that needs to be deleted.

**Pull request merged**